pkg/storage/s3/s3.go

```diff
@@ -344,14 +344,16 @@ func (d *driver) CreateStorage(cr *opapi.ImageRegistry) error {
 func (d *driver) RemoveStorage(cr *opapi.ImageRegistry) error {
 	if !cr.Status.Storage.Managed {
 		return nil
+	} else if len(cr.Status.Storage.State.S3.Bucket) == 0 {
+		return fmt.Errorf("unable to remove storage, no bucket name available in image registry resource")
```

Comment:
is this going to block finalization?  I don't think we want to block finalization over this.

Reply:
good point, updated